middleware/2022-10-04-1.3.40.md

```diff
@@ -32,15 +32,17 @@ We have modified the DSFINV-K export so that non pos-receipts (e.g. zero receipt
 - _fiskaltrust.Middleware.Queue.SQLite v1.3.40_
 
 ## Feature: Queue migration timeout is now configurable
-We have made it possible to configure the migration timeout duration for queues by adding the key-value pair parameter MigrationsTimeoutSec to the queue configuration. In some cases when performing a migration for big queues the default timeout was not long enough leading to some errors.
+
+We have made it possible to configure the migration timeout duration for queues by adding the key-value pair parameter `MigrationsTimeoutSec` to the queue configuration. In some cases when performing a migration for big queues the default timeout was not long enough leading to some errors.
+
 
 ### Affected packages:
 - _fiskaltrust.Middleware.Queue.EF v1.3.40_
 - _fiskaltrust.Middleware.Queue.MySQL v1.3.40_
 - _fiskaltrust.Middleware.Queue.SQLite v1.3.40_
 
 ## Feature: TAR export modi for the queue
-We have added several TAR export modi for the queue for the TAR-File export during the daily closing to avoid a growth of the database due to open transactions. We noticed that more and more database size were increasing leading to issues in terms of perfomance and data upload.To solve this issue we implemented a queue parameter allowing to configure different modi for the TAR-File export:
+We have added several TAR export modi for the queue for the TAR-File export during the daily closing to avoid a growth of the database due to open transactions. We noticed that more and more database size were increasing leading to issues in terms of perfomance and data upload.To solve this issue we implemented a queue parameter`TarFileExportMode` allowing to configure different modi for the TAR-File export:
```

Comment:
```suggestion
We have added several TAR export modi for the queue for the TAR-File export during the daily closing to avoid a growth of the database due to open transactions. We noticed that more and more database size were increasing leading to issues in terms of perfomance and data upload.To solve this issue we implemented a queue parameter `TarFileExportMode` allowing to configure different modi for the TAR-File export:
```